roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -32,7 +32,7 @@ discovery:
 
 gateway:
   expected_master_nodes: ${NODE_QUORUM}
```

Comment:
We're bringing the ability to have client nodes with https://github.com/openshift/openshift-ansible/pull/3509

So I would like to keep this in there, it doesn't do any harm being there now, correct?